src/test/java/com/rocketden/main/testutil/TestUtility.java

```diff
@@ -1,8 +1,8 @@
-package com.rocketden.main.util;
+package com.rocketden.main.testutil;
 
 import com.google.gson.Gson;
 
-public class Utility {
+public class TestUtility {
```

Comment:
Is there any reason this needs to be in a `testutil` package rather than `util`? Since you renamed it I thought that would fix the conflict, but I may be wrong on this so lmk

Reply:
Good point -- fixed 👍